11-node-frameworks/classworks/Podgornaya_Katerina/index.js

```diff
@@ -0,0 +1,63 @@
+const Koa = require('koa');
+const Router = require('koa-router');
+const serve = require('koa-static');
+const bodyParser = require('koa-bodyparser');
+const joi = require('joi');
+let session = require('koa-generic-session');
+
+const app = new Koa();
+const router = new Router();
+app.keys = ['some-secret-key-one', 'some-secret-key-two'];
+const data = [];
+
+const schema = joi.object({
+    lastname: joi.string().required(),
+    description: joi.string().min(3).required(),
+});
+
+const get = (context) => {
```

Comment:
keep in mind that counter should work for any request to the server, even posting form. Split counting and getting the "session counter"

Reply:
basically it should be middleware which works for each request to the server, without any response

Reply:
Still valid point

Reply:
i added middleware before routing, so for every request to server counter works and counter number you can see at '/get' request:

my middleware:
.use(async (context, next) => {
    setCounter(context);
    await next();
  })

is it wrong?

![image](https://user-images.githubusercontent.com/52997832/89279833-81b3e680-d650-11ea-9d43-43396fdd1254.png)